include/picongpu/plugins/openPMD/restart/LoadSpecies.hpp

```diff
@@ -206,15 +208,19 @@ namespace picongpu
                     }
                 }
 
-                // Note: the global domain offset is already included in params->window.globalDimensions.offset
-                DataSpace<simDim> const patchOffset
-                    = params->window.globalDimensions.offset + params->window.localDimensions.offset;
+                const SubGrid<simDim>& subGrid = Environment<simDim>::get().SubGrid();
+                auto globalDOmainOffset = subGrid.getGlobalDomain().offset;
+                /* window.localDimensions.offset is in PIConGPU for a restart zero but to stay generic we take
+                 * the variable into account.
+                 */
+                DataSpace<simDim> const patchTotalOffset = globalDOmainOffset + params->window.globalDimensions.offset
```

Comment:
Maybe must be `DataSpace<simDim> const patchTotalOffset = globalDOmainOffset + subGrid.getLocalDomain()`